docs/recipes/using-external-config-file.md

```diff
@@ -30,10 +30,11 @@ Beneficial because it's keeping tasks DRY and config.json can be used by another
 ###### `gulpfile.js`
 
 ```js
-// npm install --save-dev gulp gulp-uglify
+// npm install --save-dev gulp gulp-uglify merge-stream
```

Comment:
just checked the amount of downloads on each. I'm going to switch it back to merge-stream